.github/workflows/black_and_test.yml

```diff
@@ -18,7 +18,13 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        python setup.py install
```

Comment:
Following [`pytest`'s best practices for integration with `setuptools`](http://doc.pytest.org/en/3.0.2/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner), this might be `pip install .` instead.